packages/accounts-base/accounts_server.js

```diff
@@ -176,9 +176,10 @@ Ap._failedLogin = function (connection, attempt) {
   });
 };
 
-Ap._successfulLogout = function () {
+Ap._successfulLogout = function (connection, userId) {
+  const user = userId && this.users.findOne(userId);
```

Comment:
(Moreover, you are fetching the user object with all fields, which can potentially be large.)